pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[build-system]
+requires = ["setuptools>=41", "wheel", "setuptools-git-versioning"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name="micro-manager-precice"
+dynamic = [ "version" ]
+dependencies = [
+    "pyprecice>=3.0.0.0", "numpy", "mpi4py", "scikit-learn"
+]
+requires-python = ">=3.8"
+authors = [
+    { name = "The preCICE Developers",  email="info@precice.org"}
+]
+maintainers = [
+    { name = "Ishaan Desai",  email="ishaan.desai@ipvs.uni-stuttgart.de"}
+]
+description="A tool which facilitates two-scale macro-micro coupled simulations using preCICE."
+readme = "README.md"
+license={ text = "GPLv3" }
+keywords = [ "preCICE", "multiscale", "coupling" ]
+classifiers=[
+"Development Status :: 4 - Beta",
+"License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+"Programming Language :: Python :: 3 :: Only",
+"Programming Language :: Python :: 3.8",
+"Topic :: Scientific/Engineering",
+]
+
+[project.urls]
+Homepage = "https://precice.org"
+Documentation = "https://precice.org/tooling-micro-manager-overview.html"
+Repository = "https://github.com/precice/micro-manager"
+"Bug Tracker" = "https://github.com/precice/micro-manager/issues"
+
+[project.scripts]
+micro_manager_precice = "micro_manager.micro_manager:main"
```

Comment:
Is the name change of the executable intentional?

Reply:
Yes, I want to make it consistent with the package name. In any case, this name should not be directly used anywhere as per my understanding.

Reply:
A tutorial uses it, but this can be fixes easily.

Reply:
> A tutorial uses it, but this can be fixed easily.

Yes, right. Actually using the executable is the best way to use the tool, so I now made it the default way to use the Micro Manager. I will open another pull request in the tutorials repo to change the name.